pkg/minishift/openshift/oc.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package openshift
+
+import (
+	instanceState "github.com/minishift/minishift/pkg/minishift/config"
+	"github.com/minishift/minishift/pkg/util"
+)
+
+// runner executes commands on the host
+var runner util.Runner = &util.RealRunner{}
```

Comment:
Sorry, should have commented clearly.

I mean to say this `var runner = &util.RealRunner{}` , We can omit datatype during variable definition.